tests/ci/build-release.yaml

```diff
@@ -81,7 +81,7 @@ steps:
     ./build/venv/v/bin/vtools build redpanda-dashboard --conf=vtools/vtools/artifacts/ci/vtools-$_COMPILER-$_BUILD_TYPE.yml
     ./build/venv/v/bin/vtools build pkg --conf=vtools/vtools/artifacts/ci/vtools-$_COMPILER-$_BUILD_TYPE.yml --format=rpm --format=deb --format=tar
     ./build/venv/v/bin/vtools publish --conf=vtools/vtools/artifacts/ci/vtools-$_COMPILER-$_BUILD_TYPE.yml
-    ./build/venv/v/bin/vtools ci github-event --event-type packages-created || true # let's not mess with the release pipeline yet
+    ./build/venv/v/bin/vtools ci github-event --event-type release-uploaded || true # let's not mess with the release pipeline yet
```

Comment:
not directly related to this PR, but `vtools ci github-event` currently reads `GH_TOKEN`, but we now define `VTOOLS_GITHUB_TOKEN`, right? So we might need to update the vtools repo

Reply:
Those are in different pipelines. in GCB we create `GH_TOKEN` from the already declared `_GITHUB_API_TOKEN`

Reply:
I see. makes sense